tests/x11/chromium.pm

```diff
@@ -27,27 +23,19 @@ sub run {
     ensure_installed 'chromium';
 
     # avoid async keyring popups
-    x11_start_program('chromium --password-store=basic', target_match => 'chromium-main-window', match_timeout => 50);
-
-    wait_screen_change { send_key 'esc' };    # get rid of popup (or abort loading)
-    send_key 'ctrl-l';    # select text in address bar
+    # allow key input before rendering is done, see poo#109737 for details
+    x11_start_program('chromium --password-store=basic --allow-pre-commit-input', target_match => 'chromium-main-window');
 
-    # Additional waiting to prevent unready address bar
-    # https://progress.opensuse.org/issues/36304
-    assert_screen 'chromium-highlighted-urlbar';
-    type_address('chrome://version ');
-    assert_screen 'chromium-about';
-
-    send_key 'ctrl-l';
-    assert_screen 'chromium-highlighted-urlbar';
     type_address('https://html5test.opensuse.org');
-    assert_screen 'chromium-html-test', 90;
+    assert_screen('chromium-html-test');
 
     # check a site with different ssl configuration (boo#1144625)
-    send_key 'ctrl-l';
-    assert_screen 'chromium-highlighted-urlbar';
     type_address('https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg');
-    assert_screen 'chromium-opensuse-logo', 90;
+    assert_screen('chromium-opensuse-logo');
```

Comment:
Originally I deleted the timeout when changing to rely on x11_start_program because it defaults to 90s. The default here is 30s but it still seems to work fine so I figured if we can reduce it that's even better